README.md

```diff
@@ -1,25 +1,78 @@
-# User Requirements for Github Repository Investment Project
-To assess investment potential of all Github repositories, the client would like to be able to identify fast growing repositories as well as analyze a specific repository.
+# Overview
+This project aims to assess the investment potential of all Github repositories. The goal is to be able to identify fast growing repositories as well as analyze a specific repository. The data will be delivered in the format of a SQL database which will update at a daily frequency.
 
-## Metrics
+# Metrics
 
-The metrics/data points that the client would like to analyze is as follows:
+The metrics/data points we would like to analyze are as follows:
 
 1. Growth rate of stars
 2. Growth rate of commits
 3. Custom Analysis
 
-The client will use the growth rate of stars and growth rate of commits as metrics to decide which repositories to invest in. The client would also like to be able to do Ad hoc custom analysis to conduct specific, tailored analyses to assess investment potential. The final product needs to be flexible enough to provide this functionality.
+The growth rate of stars and growth rate of commits are the metrics we will use to decide which repositories to invest in. The final product also needs to have the functionality to conduct Ad hoc custom analysis. We can better assess investment potential with specific, tailored analyses.
```

Comment:
Suggestion: Consider removing the investment mention, as once delivered, your repo can be used as a product by anyone who wishes to measure stars and commits (for any reason including but not limited to investment). Makes it look more pro! ✨